From 6087b30a65d1a742ff0ff75f00aedd39ded33bf6 Mon Sep 17 00:00:00 2001
From: Nicola Sella <nsella@redhat.com>
Date: Wed, 7 Dec 2022 16:53:58 +0100
Subject: [PATCH 2/3] Revert "Add complex_deps requirement to choice1b
 testcase"

This reverts commit 9cb62e9db34cb62031cc5b0a444b4bff16d39db9.
---
 test/testcases/choicerules/choice1b.t | 1 -
 1 file changed, 1 deletion(-)

diff --git a/test/testcases/choicerules/choice1b.t b/test/testcases/choicerules/choice1b.t
index cf6051ce..fc47b722 100644
--- a/test/testcases/choicerules/choice1b.t
+++ b/test/testcases/choicerules/choice1b.t
@@ -1,4 +1,3 @@
-feature complex_deps
 repo system 0 testtags <inline>
 #>=Pkg: B 1 1 noarch
 #>=Prv: P = 1
-- 
2.38.1

